Sources/LiCore/Models/LiGenericQueryResponse.swift

```diff
@@ -15,7 +15,7 @@
 import Foundation
 /// Model for return object for the getGenericNoLiqlClient
 public struct LiGenericQueryResponse: LiBaseModel {
-    let data: [String: Any]
+    public let data: [String: Any]
```

Comment:
why though?

Reply:
It's a model, the data field should be accessible outside the SDK, hence.